README.md

```diff
@@ -98,6 +98,12 @@ The application is running on port `3000`
 This site reflects the documentation at the point where the command was written. Viewing changes to
 the OpenAPI documentation requires restarting the development server).
 
+### Schema changes
+If you make changes to any of the schema files in `src/schema`, it will be necessary to generate migration files in order for the changes to be reflected in the database.
```

Comment:
```suggestion
If you make changes to any database schema in `src/schema`, it will be necessary to generate migration files in order for the changes to be reflected in the database. Database schemas are indicated by their use of the drizzle `pgTable` method. (Changes to the Zod "entity" schemas do not require database migrations)
```

Reply:
It seems there has been some confusion on when migrations are and are not required. I hope this change will help clarify the distinction and document it for the future.

Reply:
this is great!